src/client/pages/AssessmentSection/DataTable/Table/TableBody/TableBody.tsx

```diff
@@ -13,10 +30,115 @@ type Props = {
   data: TableData
 }
 
+type UseValidateProps = {
+  table: Table
+  rowsData: TypeRow[]
+}
+// ===================
+
+const useValidations = (props: UseValidateProps) => {
```

Comment:
this hook looks nice. I would move it in a separate file, under subfolder  hooks.
question, can we use this hook in Table, not TableBody?